source/read-write-concern/tests/operation/default-write-concern-3.2.yml

```diff
@@ -8,7 +8,6 @@ data:
   - {_id: 2, x: 22}
 collection_name: &collection_name default_write_concern_coll
 database_name: &database_name default_write_concern_db
-other_collection_name: &other_collection_name other_collection_name
```

Comment:
Just to confirm, this was never used within this file and just copypasted across all of the new tests in your last PR? Likewise for `default-write-concern-2.6.yml`.

Reply:
Correct.